availability/storage.py

```diff
@@ -29,19 +29,18 @@ def get_elasticsearch(check_availability=False):
     """Return Elasticsearch instance.
```

Comment:
This method must raise Exception. It's very important. 

Code that is using this method should handle exception